app/assets/javascripts/controllers/dialog_user/dialog_user_controller.js

```diff
@@ -0,0 +1,25 @@
+ManageIQ.angular.app.controller('dialogUserController', ['API', 'serviceTemplateId', 'serviceTemplateCatalogId', function(API, serviceTemplateId, serviceTemplateCatalogId) {
+  var vm = this;
+
+  vm.$onInit = function() {
+    return new Promise((resolve) => {
+      resolve(API.get('/api/service_templates/' + serviceTemplateId + '/service_dialogs?expand=resources&attributes=content').then(init));
```

Comment:
when possible, might be a good idea to use template literals http://es6-features.org/#StringInterpolation. Like 6 is a gooooooood candidate 😋